docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: 'StandaloneOSX'
+          context: unity
+```
+
+##### Other Platforms
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      # WebGL
+      - unity/build:
+          name: 'build-webgl'
+          step-name: 'Build WebGL'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'webgl'
+            editor_version: '2021.3.1f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: 'WebGL'
+          compress: false
+          context: unity
+
+      # Android
+      - unity/build:
+          name: 'build-android'
+          step-name: 'Build Android'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'android'
+            editor_version: '2021.3.2f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: 'Android'
+          context: unity
+
+      # iOS
+      - unity/build:
+          name: 'build-ios'
+          step-name: 'Build iOS'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'ios'
+            editor_version: '2021.3.2f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: 'iOS'
+          context: unity
+
+      # tvOS
+      - unity/build:
+          name: 'build-tvOS'
+          step-name: "Build Apple's tvOS"
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/windows-2019'
+            size: 'large'
+            editor_version: '2021.3.2f1'
+            target_platform: 'appletv'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: tvOS
+          compress: true
+          context: unity
+```
+
+#### Professional License
+
+Below you will see how the build job can be used to build your project for several platforms using
+Unity's professional license. If you are unsure what this means, navigate to the
+[Activation](/docs/circleci/activation) section. All examples are based on CircleCI's
+[Demo Project](https://github.com/CircleCI-Public/Unity2D-Demo-Game-CI-CD).
```

Comment:
```suggestion
This examples shows how to build your project for several platforms using a Pro License.
```

Reply:
Be careful not to copy paste too much, repetitive texts cause people to stop reading what you write and it's very prone to bugs.

Reply:
Same as above.